docs/model.md

```diff
@@ -1,16 +1,49 @@
 # Model
 
-OTG is an actively developing specification, with contributions coming from real [use cases](/examples/#use-cases). On high level, the model allows to express the following building blocks of a traffic generator configuration:
+## Formal Model
+The formal [model specification](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) can be found on GitHub under [Open Traffic Generator](https://github.com/open-traffic-generator) organization. To explore the model, a viewer friendly [**ReDoc rendering**](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml) is available as well. The OTG APIs support both REST and gRPC interfaces.
+
+## Building Blocks
+
+OTG is an actively developing specification, with contributions from real [use cases](/examples/#use-cases). The model allows to express the following building blocks of a traffic generator configuration:
+
+* **Test Ports** with Layer 1&2 capabilities, including:
+	- LLDP, LAG, LACP
+* Emulated **OTG Devices** with Layer 2&3 features:
```

Comment:
Again "Emulated OTG Device" or "Emulated Device"?  I prefer the latter as it seems more consistent with "Test Ports" and "Traffic Flows" (no OTG in those).  I'm ok either ways.  If you change it in one place, then change everywhere to keep it consistent.

Reply:
Done by using "Emulated Devices"